scripts/check_system_python.py

```diff
@@ -135,3 +137,20 @@
             raise Exception(
                 "The package `pylint` isn't installed in the virtual environment."
             )
+
+        # Install the package (`numpy`).
+        logging.info("Installing the package `numpy`.")
+        subprocess.run(
+            [uv, "pip", "install", "numpy", "--system"],
```

Comment:
This is missing the `--break-system-packages` flag which can be included with 

```suggestion
            [uv, "pip", "install", "numpy", "--system"] + allow_externally_managed,
```

This is causing your failure on alpine.